src/main/java/com/todayhouse/domain/story/application/StoryReplyServiceImpl.java

```diff
@@ -46,9 +51,23 @@ public void deleteReply(User user, ReplyDeleteRequest request) {
 
     @Override
     @Transactional(readOnly = true)
-    public Page<ReplyGetResponse> findReplies(Long storyId, @PageableDefault Pageable pageable) {
+    public Page<ReplyGetResponse> findReplies(User user, Long storyId, @PageableDefault Pageable pageable) {
         Page<StoryReply> storyReplies = replyRepository.findByStoryId(storyId, pageable);
-        return storyReplies.map(r -> new ReplyGetResponse(r.getId(), r.getContent(), r.getCreatedAt(), r.getUser()));
+
+        Page<ReplyGetResponse> map = storyReplies.map(r -> new ReplyGetResponse(r.getId(), r.getContent(), r.getCreatedAt(), r.getUser()));
+        Long userId;
+
+        if (user == null) {
+            userId = null;
+        } else {
+            Optional<User> byEmail = userRepository.findByEmail(user.getEmail());
+            User user1 = byEmail.orElseThrow();
+            userId = user1.getId();
+        }
+        Long finalUserId = userId;
```

Comment:
```suggestion
        Long userId = null;

        if (user != null) {
            User findUser = userRepository.findByEmail(user.getEmail()).orElseThrow(???);
            userId = findUser.getId();
        }
```
이렇게 하면 코드를 줄일 수 있습니다!